pwndbg/gdblib/disasm/arch.py

```diff
@@ -951,5 +953,52 @@ def _common_store_annotator(
 
             instruction.annotation = f"{address_str} => {self._telescope_format_list(telescope_addresses, TELESCOPE_DEPTH, emu)}"
 
+    def _common_move_annotator(self, instruction: PwndbgInstruction, emu: Emulator):
+        """
+        This function handles annotating `MOV` type instructions - where the value of one register is placed into another.
+        """
+        if len(instruction.operands) == 2:
+            left, right = instruction.operands
+            # If we already used emulation, use the result, otherwise take the source operand before_value
+            result = left.after_value or right.before_value
+            if result is not None:
+                TELESCOPE_DEPTH = max(0, int(pwndbg.config.disasm_telescope_depth))
+
+                telescope_addresses = self._telescope(
+                    result,
+                    TELESCOPE_DEPTH + 1,
```

Comment:
Sometimes we pass in +1, sometimes we pass in without +1, why?

(May be worth to add a comment, tho idk, maybe this is obvious? I don't know from just looking here)

Reply:
Also below we do without +1